website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -38,7 +38,267 @@ description: >-
 
 # Team tokens API reference
 
-Team API tokens grant access to a team's workspaces. Each team can have an API token that is not associated with a specific user. You can create and delete team tokens and list an organization's team tokens.
+Team API tokens grant access to a team's workspaces. Teams are not limited to a single token, and can have multiple tokens at a time. Team tokens are not associated with a specific user.
+
+Teams relying on the [**legacy**](/terraform/cloud-docs/api-docs/team-tokens#legacy-team-tokens-api-reference) team token API (`/teams/authentication-token`), can only create a **single**, valid token at a time. Generating a new legacy token when one already exists for the team revokes the existing legacy token, replacing it with a new team token.
```

Comment:
Should there be a team ID param in the legacy API path?

Reply:
Yup! Great catch. I've updated this ✔️